OpenMTR/code/ReadData.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.IO;
+using System.Text.RegularExpressions;
+using OpenCvSharp;
+using System.Collections.Generic;
+
+
+namespace OpenMTR
+{
+    static class ReadData
+    {
+        // Get a list of data objects built from all the images found in a directory
+        public static List<DataObject> GetDataObjectList(string DirectoryPath)
+        {
+            List<DataObject> Data = new List<DataObject>();
+            foreach (string FilePath in Directory.GetFiles(DirectoryPath))
+            {
+                if (Regex.IsMatch(FilePath, ".(jpe?g|png)$"))
+                {
+                    Data.Add(GetDataObject(FilePath));
+                }
+            }
+            return Data;
+        }
+        public static void GetDataObjectList(string DirectoryPath, List<DataObject> Data)
+        {
+            Data = GetDataObjectList(DirectoryPath);
+        }
+
+        // Get a single data object from a path to the image file
+        public static DataObject GetDataObject(string ImagePath)

```

Comment:
This could be set to `private` since I doubt we will be using this class outside the `GetDataObjectList` method.